packages/verification-vue-component/src/style/settings/_variables.scss

```diff
@@ -1,3 +1,19 @@
+
+:root {
+    --cf-verification-primary-color: #0669f2;
+    --cf-verification-secondary-btn-bg-color: #e5e3d6;
```

Comment:
Inconsistent naming, one is just called primary-color and one secondary-**btn-bg**-color, why not just secondary-color?